src/backend/base/langflow/api/utils.py

```diff
@@ -85,11 +85,13 @@ def get_is_component_from_data(data: dict):
 
 
 async def check_langflow_version(component: StoreComponentCreate):
-    from langflow.version.version import __version__ as current_version  # type: ignore
+    from langflow.version import get_version
+    __version__ = get_version()
 
     if not component.last_tested_version:
-        component.last_tested_version = current_version
+        component.last_tested_version = __version__
 
+    # TODO: FRAZ - Figure out what I want the behavior to be for nightly builds
```

Comment:
TODO:  Based on the code, I _suspect_ we are just logging a warning that will say: 
```
Your version of langflow (<some nightly version>) is outdated. Please update to <latest stable>
```

Which, doesn't really make sense - we probably want it to say "there's a new nightly build to try" or even just ignore this since it's already known you're on a nightly build, but low priority either way I believe. Not a blocker for this PR.

Reply:
I'd say we just ignore it